alf/environments/suite_dmc.py

```diff
@@ -161,14 +108,18 @@ def dmc_loader(environment_name,
         seed=seed,
         visualize_reward=False,
         from_pixels=from_pixels,
-        episode_length=max_episode_steps,
-        height=pre_transform_image_size,
-        width=pre_transform_image_size,
+        episode_length=max_episode_steps * 2,  #infinite
```

Comment:
add a comment about *2. Otherwise, people will scratch their head to understand it,